### PR TITLE
update estimate_wct to handle rotd in im_calc

### PR DESCRIPTION
**Pull request by shot0829**
#### Description
https://wiki.canterbury.ac.nz/display/QuakeCore/IM_calc+WCT+Estimation+update
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Review by jasonmotha — DISMISSED** [review]

**Review by sungeunbae — APPROVED** [review]

**Review by jasonmotha — DISMISSED** [review]

**Review by jpa198 — APPROVED** [review]

File: shared_workflow/platform_configs/platform_local.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "PLATFORM_ACCOUNTS": ["nesi00213", "comm00213"],
   "DEFAULT_MEMORY" : "16G",
   "SCHEDULER": "bash",
+  "VM_PERT_DEFAULT_NCORES":1,
```
Comment (jasonmotha) [inline shared_workflow/platform_configs/platform_local.json:15]: It's default set to 4 in the slurm script. Should we keep that?

Comment (shot0829) [reply shared_workflow/platform_configs/platform_local.json:15]: this is for local runs, which i doubt anyone would use?

Comment (shot0829) [reply shared_workflow/platform_configs/platform_local.json:15]: and all other values are set to 1 as well.

Comment (shot0829) [reply shared_workflow/platform_configs/platform_local.json:15]: i personally think its fine to keep it as one, as if anyone is using this config, that means
1. the workflow failed to identify the machine, thus safer to assume its serial
2. advanced user know what he is doing, thus should be able to adjust the config to required.

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**